src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -405,6 +359,65 @@ class StatementSegment(ansi_dialect.get_segment("StatementSegment")):  # type: i
     parse_grammar = match_grammar
 
 
+@tsql_dialect.segment(replace=True)
+class GreaterThanOrEqualToSegment(BaseSegment):
+    """Greater than or equal to operator.
+
+    N.B. Patching to add !< and
+    to allow spaces between operators.
```

Comment:
Is this part of the comment still accurate?
>to allow spaces between operators.

(Maybe so, wasn't sure since I don't see anything about `allow_gaps` here.)

Reply:
Yeah its the lack of the `allow_gaps` operator here that allows the gaps since its True by default. The relevant T-SQL test fixtures pass 😄 

example:
![image](https://user-images.githubusercontent.com/80432516/147858431-0defa9d3-4069-4af5-aad3-4e5236df0a22.png)